src/plugins/plot/chart/MCTChartSeriesElement.js

```diff
@@ -130,6 +131,7 @@ export default class MCTChartSeriesElement {
   reset() {
     this.buffer = new Float32Array(20000);
     this.count = 0;
+    //TODO: Should we call resetYOffsetAndSeriesDataForYAxis here?
```

Comment:
Do we need to create an issue to follow up on this?

Reply:
I need to investigate this a bit. I'll create an issue for it.